deepspeed/checkpoint/utils.py

```diff
@@ -5,7 +5,7 @@
 
 import os
 import torch
-from .constants import (MODEL_FILE_PREFIX, MODEL_FILE_SUFFIX, OPTIM_FILE_SUFFIX, ZERO_FILE_PREFIX)
+from .constants import (MODEL_FILE_PREFIX, MODEL_FILE_SUFFIX, OPTIM_FILE_SUFFIX, ZERO_FILE_PREFIX, UNIVERSAL_CHECKPOINT_INFO, UNIVERSAL_CHECKPOINT_VERSION_KEY, UNIVERSAL_CHECKPOINT_VERSION_VALUE)
```

Comment:
FYI @xylian86 - can you run the precommit formatter on this branch so it will pass our Formatting check?

```
pre-commit run --all-files
```